PyInstaller/utils/hooks/__init__.py

```diff
@@ -984,3 +986,61 @@ def relpath_to_config_or_make(filename):
     # Relative path in the dist directory.
     prefix = _find_prefix(filename)
     return os.path.relpath(os.path.dirname(filename), prefix)
+
+
+def get_typelibs(module, version):
+    datas = []
+    statement = """
+import gi
+gi.require_version("GIRepository", "2.0")
+from gi.repository import GIRepository
+print(GIRepository.Repository.get_search_path())"""
+    typelibs_path = eval_statement(statement)[0]
+    pattern = os.path.join(typelibs_path, module + '*' + version + '*')
+    for f in glob.glob(pattern):
+        datas.append(gir_library_path_fix(f))
```

Comment:
That's a good question. Some hooks print warning (e.g. `hook-django.py`). So also print warning.